archunit/src/test/java/com/tngtech/archunit/lang/ArchRuleTest.java

```diff
@@ -41,6 +42,7 @@
     @Before
     public void setUp() {
         ignoreFile().delete();
+        ArchConfiguration.get().reset();
```

Comment:
There is a rule for that, that automatically sanitizes the configuration: `ArchConfigurationRule`
Then you don't need to manually reset the configuration, etc. You can check out the usage in other tests :slightly_smiling_face:

Reply:
Ah, cool. Didn't know that 👌🏽